docs/testplan/BGP-BGPSentinel.md

```diff
@@ -0,0 +1,51 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+## Overview
+The purpose is to test that BGPSentinel (BGPS) host will setup an IBGP connection to FRR. BGPSentinel (BGPS) will advertise and withdraw routes to FRR. As BGPSentinel session is IBGP and not directly connected to the DUT, this feature relies on V4 and V6 nht resolve via default configuration in FRR.
+
+### Scope
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is not to test specific API, but functional testing of BGPSentinel configuration on SONIC system.
+
+### Testbed
+The test will run on the following testbeds:
+* t1-lag (vs)
+
+## Setup configuration
+This test doesn't require any configuration in ansible deployment. IBGP session between BGPSentinel (BGPS) host and DUT are created in set up phase of this case and are cleaned in tear down phase of this case. BGPSentinel (BGPS) host is simulated by exabgp in PTF.
+
+## Test
```

Comment:
- Please include the topology which you shared during community review in the testplan document

Reply:
The topology added to the testplan doc. Thanks.